### ISSUE TITLE
Potential bug on 1.2.0

### ISSUE THREAD

**Issue by martindusinberre** [henokemp/lit-feelingthepast#3]
@henokemp I just want to be clear that this is the page we're having problems transitioning away from, right?
If so, one issue may be the way that the page is oriented. Can you give me a sense of how you would like the layout to be? If it's too difficult to describe, draw it old-style on a piece of paper and attach the file here so I can see. 
I've now forked your repository so I can keep looking through that and see if anything comes up.

**Comment by henokemp** [comment]
@martindusinberre Correct, upon reloading the page the transition will be done. But not by just pushing on the "Enter the basement" button. Keep in mind though, that Marugoto currently displays an outdated version of _Feeling the Past_. You'll find the general layout concept attached. 1.2.0 should be considered an "Exposure Page" with an added picture of a painting. So on the left side there's an environmental imageComponent, on the right side there's the textComponent and below them both is the added painting, centered on the webpage. 

[page-layout-15.4.20.pdf](https://github.com/henokemp/lit-feelingthepast/files/4546121/page-layout-15.4.20.pdf)

**Comment by martindusinberre** [comment]
@henokemp great. Can you just give me a sense of how the Choice/Exercise/Exposure pages relate to the numbered pages on your repo?

**Comment by martindusinberre** [comment]
(I'm working off the json files, not the test environment.)

**Comment by henokemp** [comment]
Choice: 1.0.0, 1.0.1, 1.0.2, 1.4.0
Exposure: 1.1.0, 1.1.1, 1.2.0, 1.2.2, 1.3.1, 1.5.0, 1.5.2
Exercise: 1.2.1, 1.3.0, 1.4.1, 1.4.2, 1.4.3, 1.5.1

In general: 

**Choice pages** have multiple transition-possibilities from which the player has to choose one (indicator: multiple pageTransitions) and (in general) 1 environmental imageComponent and 1 textComponent. 

**Exposure pages** have 1 pageTransition and (in general) 1 environmental imageComponent and 1 textComponent.

**Exercise pages** have 1 textExercise, which should be completed to proceed, (in general) 1 environmental imageComponent and 1 textComponent and depending on the exercise multiple additional imageComponents.

**Comment by martindusinberre** [comment]
As a test to see whether the 'bug' might be from the positioning of the images, please follow my redesign suggestions (#5 ) and we can see if this addresses this particular problem
